stable/launcher/powershell-wrapper

```diff
@@ -45,11 +52,21 @@ TestDistro(){
     # Tests for unsupported distrobutions go here
     if [ "${DIST}" = "fedora" -a "${REV}" = "28" ] ; then
         ReportUnsupported -D $DIST -R $REV
-        # Workaround for https://github.com/powershell/powershell/issues/6132
-        export TERM=xterm
+        echoerr "Use 'powershell-preview' package for support."
+        WorkaroundTerminalIssue
+    elif [ "${DIST}" = "manjaro" ] ; then
+        ReportUnsupported -D $DIST -R $REV
+        WorkaroundTerminalIssue
+    elif [ "${DistroBasedOn}" = "arch" ] ; then
```

Comment:
same question about `DIST` vs `DistroBasedOn`

Reply:
I don't have an Arch machine to get the DIST value.  The potential issue here is this section of logic might overreach, but it would be another unsupported distro, so I don't think it's an issue.

Reply:
I verified the logic works in a container:
`Warning: PowerShell 6.0 does not support arch - and only community support is available!`